Modules/CodeTrek/Resources/views/index.blade.php

```diff
@@ -157,7 +157,7 @@ class="btn-sm btn-primary mr-1 text-decoration-none"
                                 </td>
                                 <td>
                                     @php
-                                        $daysInCodetrek = now()->diffInDays($applicant->start_date);
+                                        $daysInCodetrek = $applicant->getDaysInCodetrekAttribute($applicant);
```

Comment:
When you create accessor it can be easily be called like this
```suggestion
                                        $daysInCodetrek = $applicant->days_in_codetrek;
```